pkg/kube/controllers/boshdeployment/deployment_reconciler.go

```diff
@@ -28,8 +31,8 @@ import (
 // JobFactory creates Jobs for a given manifest
 type JobFactory interface {
 	VariableInterpolationJob(manifest bdm.Manifest) (*qjv1a1.QuarksJob, error)
-	InstanceGroupManifestJob(manifest bdm.Manifest) (*qjv1a1.QuarksJob, error)
-	BPMConfigsJob(manifest bdm.Manifest) (*qjv1a1.QuarksJob, error)
+	InstanceGroupManifestJob(manifest bdm.Manifest, linkSecrets map[string]string) (*qjv1a1.QuarksJob, error)
+	BPMConfigsJob(manifest bdm.Manifest, providerSecrets map[string]string) (*qjv1a1.QuarksJob, error)
```

Comment:
Provider secrets reads ambiguous to me. Could be a cloud provider.

Reply:
Renamed